{{cookiecutter.project_slug}}/clients/react/src/pages/LogIn.tsx

```diff
@@ -102,7 +104,7 @@ export default function LogIn() {
               <></>
             )}
           </FormControl>
-          <Button onClick={handleLogin}>Sign In</Button>
+          <Button data-cy="submit" onClick={handleLogin}>Sign In</Button>
```

Comment:
React apps now have Cypress tests.
I was hoping to skip this but having Cypress was kinda critical for testing this PR.
Also it would have been harder to have one bootstrapped app using Cypress and the other not having it.